syntax = "proto3";

// Build Steps :
// python3 -m venv env && source env/bin/activate
// python3 -m pip install grpcio grpcio-tools
// python3 -m grpc_tools.protoc -I. --python_out=. --grpc_python_out=. magnet.proto

service Magnet {
    // Sends a greeting
    rpc GetMagnet (MagnetRequest) returns (MagnetReply) {}
  }
  
// The request message containing the user's name.
message MagnetRequest {
string imdb_id = 1;
}

// The response message containing the greetings
message MagnetReply {
string url = 1;
}